make/autoconf/build-aux/config.guess

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 #
-# Copyright (c) 2012, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2012, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
There's an extra, unnecessary blank line in this file relative to upstream, that I would have liked to see removed, but I won't hold this up for that.